demos/crossroad_camera_demo/crossroad_camera_demo.hpp

```diff
@@ -42,7 +43,8 @@ static const char utilization_monitors_message[] = "Optional. List of monitors t
 
 
 DEFINE_bool(h, false, help_message);
-DEFINE_string(i, "cam", video_message);
+DEFINE_string(i, "", input_message);
+DEFINE_bool(p, false, loop_input_message);
```

Comment:
I wanted to introduce a convention to use only one char keys. That would resolve a conflict about `-` and `--` for python demos where we can set different names for a key. But since gflags automatically enables `-key` and `--key` I keep the initial name.

Reply:
May be its better to rename into `--loop` so we don't need to choose between `-` and `_` as word separator in python.

Reply:
IMO, the separator issue is unavoidable. We can't contort every option into one word. We just need to define a policy and stick to it.

About `--loop` in particular - I don't immediately see anything wrong with that name, but I didn't think this through, and for the sake of minimizing changes in one PR I think you should stick with the current name for now.

Reply:
We will have changes any way because it was `loop_video` instead of input. I think it's a good moment to make a decision to use `-loop`.

Reply:
Fair enough. Yeah, I guess we can go with `-loop`.

Reply:
Updated